tests/programs/custom/debug_test_trigger/debug_test_trigger.c

```diff
@@ -1611,27 +1655,23 @@ int test_register_access(void) {
   __asm__ volatile (R"(li    s1,     0x0
                        csrw  tinfo,  s1
                        csrr  s0,     tinfo
-                       li    s1,     0x8064
+                       li    s1,     0x01008064
```

Comment:
I don't know this CSR well enough, but I trust that it was selected well and that the ISS would help us find any mistakes if there are any.

Reply:
this changed bit is hardcoded 1 (cf. user manual)